svm/src/transaction_processor.rs

```diff
@@ -286,12 +287,12 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
                 &validation_results,
                 PROGRAM_OWNERS
             ));
+        let native_loader = native_loader::id();
+        for builtin_program in self.builtin_program_ids.read().unwrap().iter() {
```

Comment:
do we want to take this out of the timing measurement?

Reply:
It should probably go in the measurement above instead. But should not really matter. I can move it anyway.